### ISSUE TITLE
Searching public playlists

### ISSUE THREAD

**Issue by brettyj** [hechtus/squeezebox-googlemusic#66]
Hi,

Now that public playlists are officially supported in Google Music, any chance a playlist search option could be added to search for public playlists?

Thanks

**Comment by gdhbashton** [comment]
Would this also cover the 'curated' lists like the 'Making Dinner' and 'A House Party' radio stations? I'd love to be able to play those from my Squeezeserver :)

**Comment by hechtus** [comment]
Playlist search should be doable. It is already supported by gmusicapi. I'm planning to add this in the future.

**Comment by hechtus** [comment]
@gdhbashton What do you exactly mean by curated lists? Are these lists public playlists or radio stations?

**Comment by gdhgdhgdh** [comment]
(Mm, I confused my home + work GitHub accounts) Curated lists are definitely radio stations - they just have a strange interface by which to reach them:

e.g. starting at https://play.google.com/music/listen#/now, choose a category like 'Happy Hour' or 'Focusing While You Work' and then make a selection from the sub menus.

It would be fantastic to be able to browse this structure in the same way that the existing "Radio Genres" inside the Google Music menu works.

**Comment by rolandrohde** [comment]
Yes! Search and Radio already work very well, but explore/discovery is still missing in the plugin, together with Playlists, if thie could be added it would be pretty much perfect...
